src/aleph_client/conf.py

```diff
@@ -3,16 +3,20 @@
 from typing import Optional
 
 from pydantic import BaseSettings, Field
+import os
+import sys
 
 
 class Settings(BaseSettings):
+    CONFIG_HOME: Optional[str] = None
+
     # In case the user does not want to bother with handling private keys himself,
     # do an ugly and insecure write and read from disk to this file.
     PRIVATE_KEY_FILE: Path = Field(
-        default=Path("device.key"),
+        default=Path("ethereum.key"),
```

Comment:
can also be a Solana key though. I often wondered how we should solve this problem. Maybe different keys for different accounts the user creates?

Reply:
We support Ethereum by default. This is the default setting.